lib/Controller/Id4meController.php

```diff
@@ -117,6 +131,8 @@ public function __construct(
 		$this->id4MeMapper = $id4MeMapper;
 		$this->config = $config;
 		$this->l10n = $l10n;
+		$this->loginController = $loginController;
```

Comment:
Injecting a controller into another cries for just creating a shared base class instead 🙈